server/tests/src/test/java/org/infinispan/server/functional/ClusterBackupIT.java

```diff
@@ -0,0 +1,186 @@
+package org.infinispan.server.functional;
+
+import static org.infinispan.functional.FunctionalTestUtils.await;
+import static org.infinispan.util.concurrent.CompletionStages.join;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.Arrays;
+
+import org.infinispan.client.rest.RestCacheClient;
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.RestCounterClient;
+import org.infinispan.client.rest.RestEntity;
+import org.infinispan.client.rest.RestResponse;
+import org.infinispan.client.rest.RestTaskClient;
+import org.infinispan.commons.dataconversion.MediaType;
+import org.infinispan.commons.test.CommonsTestingUtil;
+import org.infinispan.commons.util.Util;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.counter.api.Storage;
+import org.infinispan.counter.configuration.Element;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
```

Comment:
Jackson databind is being killed, please avoid using it and use the internal mJson instead